site/content/docs/main/aggregator-permissions.md

```diff
@@ -0,0 +1,27 @@
+# Aggregator Permissions
+
+Through `sonobuoy gen` command you can customize multiple aspects of the run in your cluster. In these multiple aspects you can also customize aggregator permissions of a user in your cluster. These permissions act as limitations, it determines that in what tasks a user can take part in the run. We can customize this aspect of the run through `aggregator-permissions` flag.
```

Comment:
This isn't `sonobuoy gen` specific; I would just say something like
```
By default, the Sonobuoy aggregator is given very elevated permissions in order to successfully run the Kubernetes end-to-end tests.

In some situations you may want to (or need to) limit the permissions of the aggregator so that the aggregator and the pods that it creates do not have such wide-reaching permissions.

You can always customize the exact permissions of the ServiceAccount via editing `sonobuoy gen` output manually, but Sonobuoy also provides useful presets via the CLI flag, `--aggregator-permissions`.
```

Reply:
@johnSchnake Do you need these as list points or a single paragraph ?

Reply:
It can be a single paragraph. I think I just separated the thoughts there to help me think it through.